### PR TITLE
docs: updated the documentation of ceph-nfs-crd

### PR DESCRIPTION
**Pull request by parth-gr**
Added creation of NFS exports in the documentation.

Closes: https://github.com/rook/rook/issues/7064
Signed-off-by: parth-gr <partharora1010@gmail.com>

**Description of your changes:**

**Which issue is resolved by this Pull Request:**
Resolves #

**Checklist:**

- [ ] **Commit Message Formatting**: Commit titles and messages follow guidelines in the [developer guide](https://rook.io/docs/rook/master/development-flow.html#commit-structure).
- [ ] **Skip Tests for Docs**: Add the flag for skipping the build if this is only a documentation change. See [here](https://github.com/rook/rook/blob/master/INSTALL.md#skip-ci) for the flag.
- [ ] **Skip Unrelated Tests**: Add a flag to run tests for a specific storage provider. See [test options](https://github.com/rook/rook/blob/master/INSTALL.md#test-storage-provider).
- [ ] Reviewed the developer guide on [Submitting a Pull Request](https://rook.io/docs/rook/master/development-flow.html#submitting-a-pull-request)
- [ ] Documentation has been updated, if necessary.
- [ ] Unit tests have been added, if necessary.
- [ ] Integration tests have been added, if necessary.
- [ ] Pending release notes updated with breaking and/or notable changes, if necessary.
- [ ] Upgrade from previous release is tested and upgrade user guide is updated, if necessary.
- [ ] Code generation (`make codegen`) has been run to update object specifications, if necessary.

### CODE REVIEW

File: Documentation/ceph-nfs-crd.md [hunk 1]
```diff
@@ -66,6 +70,12 @@ spec:
     priorityClassName:
 ```
 
+ You can enable the creation of NFS exports in the dashboard for a given cephfs or object gateway pool with the following command in the ceph-tools container:
```
Comment (subhamkrai) [inline Documentation/ceph-nfs-crd.md:73]:
```suggestion
 You can enable the creation of NFS exports in the dashboard for a given cephfs or object gateway pool by running the following command in the toolbox container:
```

File: Documentation/ceph-nfs-crd.md [hunk 2]
```diff
@@ -66,6 +70,12 @@ spec:
...
+ 
+```yaml
```
Comment (subhamkrai) [inline Documentation/ceph-nfs-crd.md:75]:
```suggestion
```console
```

File: Documentation/ceph-nfs-crd.md [hunk 3]
```diff
@@ -66,6 +70,12 @@ spec:
     priorityClassName:
 ```
 
+ You can enable the creation of NFS exports in the dashboard for a given cephfs or object gateway pool with the following command in the ceph-tools container:
+ 
+```yaml 
+ceph dashboard set-ganesha-clusters-rados-pool-namespace <your cluster name>:<the ganesha pool name>/<the ganesha namespace>
```
Comment (subhamkrai) [inline Documentation/ceph-nfs-crd.md:76]:
```suggestion
$ ceph dashboard set-ganesha-clusters-rados-pool-namespace <pool_name>[/<namespace>]
```
see [this](https://docs.ceph.com/en/latest/mgr/dashboard/#configuring-nfs-ganesha-in-the-dashboard)

**Review by subhamkrai — CHANGES_REQUESTED** [review]

File: Documentation/ceph-nfs-crd.md (hunk 1 above)
Comment (subhamkrai) [inline Documentation/ceph-nfs-crd.md:73]: maybe this looks better
```suggestion
 To enable the creation of NFS exports in the dashboard for a given cephfs or object gateway pool by running `$ ceph dashboard set-ganesha-clusters-rados-pool-namespace <pool_name>[/<namespace>]` in the toolbox container.
```

File: Documentation/ceph-nfs-crd.md (hunk 2 above)
Comment (parth-gr) [reply Documentation/ceph-nfs-crd.md:75]: okay

File: Documentation/ceph-nfs-crd.md (hunk 3 above)
Comment (parth-gr) [reply Documentation/ceph-nfs-crd.md:76]: Yaa it's the way given in the document but I would say that's the fix I have to do for the command, [quoted here ](https://github.com/rook/rook/issues/7064#issuecomment-790819657)

Comment (parth-gr) [reply Documentation/ceph-nfs-crd.md:76]: Yaa the above command is also true for single cluster but for using multiple NFS-Ganesha clusters we require [cluster name or id](https://docs.ceph.com/en/latest/mgr/dashboard/#support-for-multiple-nfs-ganesha-clusters).

Comment (subhamkrai) [reply Documentation/ceph-nfs-crd.md:76]: in that case, you can have 2 commands one for a single cluster and the other for multiple clusters.

File: Documentation/ceph-nfs-crd.md [hunk 4]
```diff
@@ -26,7 +26,11 @@ metadata:
   namespace: rook-ceph
 spec:
   rados:
-    # RADOS pool where NFS client recovery data is stored.
+    # RADOS pool where NFS client recovery data and per-daemon configs are
+    # stored. In this example the data pool for the "myfs" filesystem is used.
+    # If using the object store example, the data pool would be
+    # "my-store.rgw.buckets.data". Note that this has nothing to do with where
+    # exported CephFS' or objectstores live.
```
Comment (BlaineEXE) [inline Documentation/ceph-nfs-crd.md:33]:
```suggestion
    # exported CephFSes or objectstores live.
```

File: Documentation/ceph-nfs-crd.md (hunk 1 above)
Comment (BlaineEXE) [reply Documentation/ceph-nfs-crd.md:73]: Good suggestion, but remove `To` from the beginning.

File: Documentation/ceph-nfs-crd.md (hunk 3 above)
Comment (BlaineEXE) [reply Documentation/ceph-nfs-crd.md:76]: An example of cluster name, ganesha pool name, and ganesha namespace that match examples we have would be good for users also.

Comment (parth-gr) [reply Documentation/ceph-nfs-crd.md:76]: ok

File: Documentation/ceph-nfs-crd.md [hunk 5]
```diff
@@ -66,6 +70,20 @@ spec:
     priorityClassName:
 ```
 
+ Enable the creation of NFS exports in the dashboard for a given cephfs or object gateway pool by running the following command in the toolbox container:
+
+[For single NFS-GANESHA CLUSTERS](https://docs.ceph.com/en/latest/mgr/dashboard/#configuring-nfs-ganesha-in-the-dashboard)
```
Comment (subhamkrai) [inline Documentation/ceph-nfs-crd.md:75]:
```suggestion
[For single NFS-GANESHA cluster](https://docs.ceph.com/en/latest/mgr/dashboard/#configuring-nfs-ganesha-in-the-dashboard)  
```

File: Documentation/ceph-nfs-crd.md [hunk 6]
```diff
@@ -66,6 +70,20 @@ spec:
...
 
+ Enable the creation of NFS exports in the dashboard for a given cephfs or object gateway pool by running the following command in the toolbox container:
+
+[For single NFS-GANESHA CLUSTERS](https://docs.ceph.com/en/latest/mgr/dashboard/#configuring-nfs-ganesha-in-the-dashboard)  
+
+```console 
+ceph dashboard set-ganesha-clusters-rados-pool-namespace <ganesha_pool_name>[/<ganesha_namespace>]
+```
+
+[FOR MULTIPLE NFS-GANESHA CLUSTERS](https://docs.ceph.com/en/latest/mgr/dashboard/#support-for-multiple-nfs-ganesha-clusters)
```
Comment (subhamkrai) [inline Documentation/ceph-nfs-crd.md:81]:
```suggestion
[For multiple NFS-GANESHA clusters](https://docs.ceph.com/en/latest/mgr/dashboard/#support-for-multiple-nfs-ganesha-clusters)
```

File: Documentation/ceph-nfs-crd.md (hunk 5 above)
Comment (subhamkrai) [reply Documentation/ceph-nfs-crd.md:75]: @leseb not sure if we need a ceph link here?

**Review by subhamkrai — CHANGES_REQUESTED** [review]

File: Documentation/ceph-nfs-crd.md (hunk 5 above)
Comment (leseb) [reply Documentation/ceph-nfs-crd.md]: I'm fine with more doc anyways :)

**Review by leseb — APPROVED** [review]

**Review by subhamkrai — APPROVED** [review]

**Pull request merged**